proto/lbm/fswap/v1/params.proto

```diff
@@ -0,0 +1,22 @@
+
+syntax = "proto3";
+package lbm.fswap.v1;
+
+option go_package = "github.com/Finschia/finschia-sdk/x/fswap/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+  // upper limit in new coin that this module can swap
+  cosmos.base.v1beta1.Coin new_coin_swap_total_limit = 1
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/Finschia/finschia-sdk/types.Coin"];
+  // constant value representing the exchange rate, without decimal (148079656)
+  uint64 swap_rate = 2;
```

Comment:
I think this value should be constant instead of Params. Bc it will never be changed.

Reply:
Yes, I remove it & set it in the InitGenesis() in next pr. Maybe I will remove it in this pr.